packaging/rpm-ostree.spec.in

```diff
@@ -4,7 +4,7 @@
 Summary: Hybrid image/package system
 Name: rpm-ostree
 Version: 2023.1
-Release: 1%{?dist}
+Release: 2%{?dist}
```

Comment:
While this is harmless (i.e. no need to update the PR and rerun CI), we don't usually bump this here because nothing reads it.

Reply:
:cry: woops